samples/README.md

```diff
@@ -195,6 +195,21 @@ Documentation for the Search API For Shopping in
   </tr>
 </table>
 
+## ![](https://www.google.com/images/icons/product/compute_engine-32.png) Cloud Compute Engine API
+
+Lets you manage the Google Compute Engine instances in your projects.
+
+Documentation for the Cloud Compute Engine API in
+[PyDoc](https://cloud.google.com/compute/docs/tutorials/python-guide).
```

Comment:
This link will redirect to new GCE GAPIC tutorial, so we should not put it here.

Reply:
README.md got replaced with the tutorial itself.